tests/Users/create-user.spec.js

```diff
@@ -0,0 +1,129 @@
+import { test, expect } from "@playwright/test";
+
+const getRandomNumber = () => {
+  return Math.floor(Math.random() * (999 - 100 + 1) + 100);
+};
+
+const wait = (seconds) => {
+  return new Promise((resolve) => {
+    setTimeout(resolve, seconds * 1000);
+  });
+};
+
+test.describe("Create user tests", () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto("./");
+    await page.getByPlaceholder("Email Address").fill("root@gmail.com");
+    await page.getByPlaceholder("Password").fill("p@s5W07d");
+    await page.getByRole("button", { name: "Log in" }).click();
+  });
+
+  test.afterEach(async ({ page }) => {
+    // await page.close();
+  });
+
+  test("Access Create User page and going back and forth", async ({ page }) => {
+    await page.getByRole('link', { name: 'Users' }).click();
+    await expect(page).toHaveURL('http://localhost:3000/users');
```

Comment:
Hardcoded URLs (http://localhost:3000/users, etc.) and credentials are found throughout the tests. It's better to use environment variables or a configuration file for such data. This makes our tests more flexible and secure, especially when running in different environments.

Reply:
fixed